### ISSUE TITLE
Full audit: Updating the ServiceNet routes on Cloud Servers created before June 3, 2013

### ISSUE THREAD

**Issue by stephamon** [rackerlabs/rackspace-how-to#1971]
Due to the 2/15 bulletin, this article may need a full audit. The bulletin offers POCs.

https://github.com/rackerlabs/rackspace-how-to/blob/master/content/cloud-servers/updating-servicenet-routes-on-cloud-servers-created-before-june-3-2013.md

**Comment by DonNateR** [comment]
A member of the Cloud Escalation team is providing the technical review for the article.

**Comment by catlook** [comment]
i'll take the api doc side of this

**Comment by stephamon** [comment]
Servers team confirmed that this can be retired.

**Comment by stephamon** [comment]
Reopening for next sprint

**Comment by stephamon** [comment]
Hey @kmsholcomb can you take a look at this article?

**Comment by kmsholcomb** [comment]
You betcha!

**Comment by kmsholcomb** [comment]
Hi @stephamon. Well, I got it in just before noon, which is morning! Anyway, I did as much streamlining as I could. I hope that those changes are OK. Let me know if you have any questions. Thanks! 

[Update ServiceNet Routes on Cloud Servers Created Before June 3 2013_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1098475/Update.ServiceNet.Routes.on.Cloud.Servers.Created.Before.June.3.2013_editsKH.pdf)
[Update ServiceNet Routes on Cloud Servers Created Before June 3 2013_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1098476/Update.ServiceNet.Routes.on.Cloud.Servers.Created.Before.June.3.2013_editsKH.docx)
